pwa/src/components/common/elementCreationNew.tsx

```diff
@@ -75,7 +75,7 @@ const ElementCreationNew: React.FC<ElementCreationNewProps> = ({ id, label, data
       )}
 
       <div className="row">
-        <h5>Create {label}</h5>
+        <span className="font-weight-bold">Create {label}</span>
```

Comment:
The input placeholder underneath says _Add_ instead of _Create_, making it look a bit odd. Maybe change the placeholder to _Create_ as well?

![image](https://user-images.githubusercontent.com/66552932/159220560-e5673a88-b1a2-4f70-9a08-a3b807a5c768.png)